packages/xo-web/src/common/intl/messages.js

```diff
@@ -1018,6 +1018,7 @@ const messages = {
   vmAffinityHost: 'Affinity host',
   vmVga: 'VGA',
   vmVideoram: 'Video RAM',
+  vmNicType: 'NIC type',
```

Comment:
@olivierlambert Should we call it NIC of PIF?

Reply:
"NIC" is the generic word for a network card, "VIF" is the right technical word for a VM virtual interface, and "PIF" is the technical word for a physical interface. So in this context, NIC is OK.